### PR TITLE
[4.0] Move featured logic from content table

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

This moves some code related to feature articles dates from `Joomla\CMS\Table\Content` to `Joomla\Component\Content\Model\Aministrator\ArticleModel`.

### Testing Instructions

Create a featured article. Set featured up/down dates.
Use batch copy to another category.
Edit the copied article. Check that featured status and featured up/down dates are copied correctly.

### Expected result

Works like before.

### Documentation Changes Required

No.

### CODE REVIEW

File: administrator/components/com_content/Model/ArticleModel.php [hunk 1]
```diff
@@ -84,15 +85,32 @@ protected function cleanupPostBatchCopy(TableInterface $table, $newId, $oldId)
...
+				->bind(':oldId', $oldId, ParameterType::INTEGER);
+
+			$featured = $db->setQuery($query)->loadObject();
+
+			if ($featured)
+			{
+				$query = $db->getQuery(true)
+					->insert($db->quoteName('#__content_frontpage'))
+					->values(
+						':newId, 0, :featuredUp, :featuredDown'
```
Comment (Quy) [inline administrator/components/com_content/Model/ArticleModel.php]: Combine to one line?

File: administrator/components/com_content/Model/ArticleModel.php [hunk 2]
```diff
@@ -84,15 +85,32 @@ protected function cleanupPostBatchCopy(TableInterface $table, $newId, $oldId)
...
+				->select(
+					[
+						$db->quoteName('featured_up'),
+						$db->quoteName('featured_down'),
+					]
+				)
+				->from($db->quoteName('#__content_frontpage'))
+				->where($db->quoteName('content_id') . ' = :oldId')
+				->bind(':oldId', $oldId, ParameterType::INTEGER);
+
```
Comment (Quy) [inline administrator/components/com_content/Model/ArticleModel.php:97]:
```suggestion
```

Comment (SharkyKZ) [reply administrator/components/com_content/Model/ArticleModel.php:97]: I added a blank in the other query instead.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on bbda926b7253ab2929c11fae9662226fab45de8a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27891">issues.joomla.org/tracker/joomla-cms/27891</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on bbda926b7253ab2929c11fae9662226fab45de8a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27891">issues.joomla.org/tracker/joomla-cms/27891</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27891">issues.joomla.org/tracker/joomla-cms/27891</a>.</sub>

**Pull request merged**